pkg/oci/build.go

```diff
@@ -148,6 +148,16 @@ func createMetadataDesc(ctx context.Context, target oras.Target, metadataFilePat
 	return defDesc, nil
 }
 
+func createEmptyDesc(ctx context.Context, target oras.Target) (ocispec.Descriptor, error) {
+	emptyDesc := ocispec.DescriptorEmptyJSON
+	emptyDesc.ArtifactType = eBPFObjectMediaType
```

Comment:
Is it required and if yes, shouldn't it be metadataMediaType?

Reply:
I think it should be even something different completly: `application/vnd.oci.empty.v1+json`

Reply:
> Is it required

Yes it is, spec says "If the config.mediaType is set to the empty value, the artifactType MUST be defined."

P.S: Empty value != no config :D 

> shouldn't it be metadataMediaType?

Great question. I wasn't too sure about it. You are right `metadataMediaType` sound better. I looked at the example 2 [here](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage) and went with the type from the layer but it might not work well if we add more layers. What do you say?

Reply:
> I think it should be even something different completly: `application/vnd.oci.empty.v1+json` (`MediaTypeEmptyJSON`)

The media type in descriptor already has this type. So it has to be something that tells about the artifact in the image. No?

Reply:
> > I think it should be even something different completly: application/vnd.oci.empty.v1+json (MediaTypeEmptyJSON)
> 
> I think that's for the mediaType.

I got confused because of the name `metadataMediaType` :see_no_evil:

Reply:
> I'm not totally sure, after reading the spec, I'd say `eBPFObjectMediaType` is better as it refers to the layer below.

I thought about this bit more and I think we should stick with `eBPFObjectMediaType`. I mean we can trim `+type` ( or use a different artifactType) but I am fine with what we have currently.